docs/resources/azure_virtual_wans.md

```diff
@@ -0,0 +1,79 @@
+---
+title: About the azure_virtual_wans Resource
+platform: azure
+---
+
+# azure_virtual_wans
+
+Use the `azure_virtual_wans` InSpec audit resource to test properties related to all Azure Virtual WANs in a subscription.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_virtual_wans` resource block returns all Azure Virtual WANs in a subscription.
+```ruby
+describe azure_virtual_wans do
+  #...
+end
+```
+
+
+## Parameters
+
+## Properties
+
+|Property            | Description                                        | Filter Criteria<superscript>*</superscript> |
+|--------------------|----------------------------------------------------|-----------------|
+| ids                | A list of the unique resource ids.                 | `id`            |
+| names              | A list of names for all the Resources.             | `name`          |
+| etags              | A list of etag for all the Resources.              | `etag`          |
+| types              | A list of types for all the resources.             | `type`          |
+| locations          | A list of locations for all the resources.         | `location`      |
+| properties         | A list of Properties all the resources.            | `properties`    |
+
+
+<superscript>*</superscript> For information on how to use filter criteria on plural resources refer to [FilterTable usage](https://github.com/inspec/inspec/blob/master/dev-docs/filtertable-usage.md).
+
+## Examples
+
+### Test that There are virtual WANs of standard type
```

Comment:
Can we give example as?

describe azure_virtual_wans do
  its('ids') { should include 'example_id' }
end

Reply:
ehh we have a better example just below .. and also [here](https://github.com/inspec/inspec-azure/pull/417/files/5d35bd3e9f8082ec18cb5e910ec2f94e78088ea7#diff-8441a2edee86ac64cf5d291138cb3ddc9dbdb79ce28ce8617d3d9147b6c730c1R73)